projects/ng-components/src/lib/controls/advanced-search/advanced-search.component.html

```diff
@@ -0,0 +1,107 @@
+<div class="flex gap-6 mb-2 px-2">
```

Comment:
δες τις ωραία buttons και dropdowns έχουμε βάλει [στον pager component](https://github.com/indice-co/Indice.Angular/blob/main/projects/ng-components/src/lib/controls/pager/pager.component.html). Μπορούμε να χρησιμοποιήσουμε το look and feel? Επίσης οι κλάσεις πρέπει να ομαδοποιηθούν με distinct names και να μπουν σε ξεχωριστό αρχείο ώστε να μπορούν να γίνουν overwrite από την εφαρμογή που θα καταναλώσει το component